### DISCUSSION TITLE
Why are all you commit messages: Added files via upload

### DISCUSSION THREAD

**Discussion by gcarreno** [Matek0611/PospoliteView#3] [General]
Hey Matek(@Matek0611),

The last time I saw this happened the person who was doing it was not using `GIT` but uploading files via GitHub's web interface.

Please tell me that you're managing your local copy of the repo with `GIT` or even a visual wrapper around `GIT` like `TortoiseGIT` or even the Desktop Client made by GitHub?

Cheers,
Gus

**Comment by Matek0611** [comment]
Hi Gus (@gcarreno),

I don't use `GIT` and I upload files via GitHub's web interface because it's easier for me now. I guess that I'll use `GIT` in the future. 

Next time I'm going to customize my commit messages. I know I should be doing this all the time, but I didn't expect my repo to become more and more recognizable, so I didn't do that.

Sorry for the inconvenience.

Regards,
Marcin

**Comment by gcarreno** [comment]
Hey Matek(@Matek0611),

Please do not apologise!!!
If I made you feel like you needed to apologise, I'm an idiot and an arse hole and I should be flogged in a public square!!!

It's alright that you use the web interface, it's just that I'm surprised that I've stumbled on 2 developers that are using it instead of using `GIT`.

May I ask why? Like, do you know how to use it but chose not to, or do you not know how to use it and you need help getting started?

The other person that was using the web interface is now a very dear friend of mine and I spent many hours guiding him on how use `GIT` in the most basic way.
He still asks me stuff like _How do I tag and release again?_, now and again, cuz it's hard to memorize. 

He chose to use the command line like me, but there are visual tools around that take the edge of the command line and the fact that it is so memory dependent, so if you ever need some help, please don't hesitate to ask!!!

Cheers,
Gus

**Comment by Matek0611** [comment]
Hi Gus (@gcarreno),

Thank you for your explanation! I know some basics but I prefer using graphical interface - dragging and dropping is faster and easier for me at this stage of the project. If I have any problems or questions related to `GIT`, I'll definitely ask you for help. Just in case, where can I ask you something? Should I use your email address provided in your profile?

Regards,
Marcin

**Comment by gcarreno** [comment]
Hey Marcin(@Matek0611),

If you prefer the GUI wrappers I'm completely Ok with that!!

Yes, please use the mail address on my profile, that why I put it there :)

If you're into Discord, I just found 3 servers that are Delphi/FPC/Lazarus based:
- Delphi Community: https://discord.gg/kCK6Kshq
- Unofficial Free Pascal: https://discord.gg/yQhwhfZS
- Delphi Programmers: https://discord.gg/fVKwVgw6

On any of them I'm `Gustavo 'Gus' Carreno` or `gcarreno` on the DM

Cheers,
Gus

**Comment by Matek0611** [comment]
Hi Gus (@gcarreno),

Thank you very much! If I have any problems, I'll text you on Discord (my profile: `Matek#8665`) 😀 

Regards,
Marcin
